_includes/head.html

```diff
@@ -45,6 +45,8 @@
 
 {% if site.enable_darkmode %}
 <!-- Dark Mode -->
+<link rel="stylesheet" href="https://gitcdn.link/repo/jwarby/jekyll-pygments-themes/master/{{ site.highlight_theme_dark }}.css" media="none" id="highlight_theme_dark" />
```

Comment:
same comment as above: it'd be great to commit this CSS file to `assets/css/` instead of loading from gitcdn.

Reply:
Alright @alshedivat,
Yes, I also have had a lot of stability issues with `gitcdn.link` in the recent weeks. Commiting the files to the css folder should make things better.

Will update with the commit as soon as I can, making all the themes available at the same time!